docker/templates/config-contracts.toml

```diff
@@ -8,7 +8,7 @@ L1_SCROLL_CHAIN_PROXY_ADDR = ""
 L1_SCROLL_MESSENGER_PROXY_ADDR = ""
 L1_ENFORCED_TX_GATEWAY_IMPLEMENTATION_ADDR = ""
 L1_ENFORCED_TX_GATEWAY_PROXY_ADDR = ""
-L1_ZKEVM_VERIFIER_V1_ADDR = ""
+L1_ZKEVM_VERIFIER_V2_ADDR = ""
```

Comment:
Note to @yiweichi, if gen-configs fails, simply remove `config-contracts.toml` and re-generate, then it should work.